src/main/kotlin/org/jitsi/nlj/rtcp/RtcpRrGenerator.kt

```diff
@@ -81,7 +81,7 @@ class RtcpRrGenerator(
             val streamStats = incomingStatisticsTracker.getSnapshot()
             val now = System.currentTimeMillis()
             val reportBlocks = mutableListOf<RtcpReportBlock>()
-            streamStats.ssrcStats.forEach { ssrc, statsSnapshot ->
+            streamStats.ssrcStats.forEach { (ssrc, statsSnapshot) ->
```

Comment:
Just curious: is this just a syntactic change, or does it produce different code?

Reply:
there was a warning in the idea and the change was automatically applied; i'd be surprised there's a change in the bytecode

Reply:
i just checked this...this was using java's `forEach`, which takes a `BiConsumer` (a lambda which takes 2 arguments) and it changed it to use kotlin's `forEach`, which takes a lambda which takes in a `Map.Entry`.  Surrounding the arguments in parentheses does destructuring so we get the individual parts of the `Map.Entry` instead of the entry itself.